recipes/funchook/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_subdirectory(src)
```

```suggestion
find_package(funchook CONFIG REQUIRED)

add_executable(${PROJECT_NAME} test_package.c)
target_link_libraries(${PROJECT_NAME} funchook::funchook}
```

Try to simplify and keep all on the same file.

---

Yeah....uh so you mean remove the contents of 'src' up to the parent, right?  And remove 'src'?

---

Yes, that`s it. Sorry, I was not clear on my comment. We usually put only a single cmakefile, conanfile and C or C++ file on test package folder and that's it. There are few exceptional cases (e.g. protobuf) where we add more files. The idea about test package is validating your generated binaries, can they link without errors? can we find the headers? and that's it.